src/Command/GetYmlMetadataCommand.cs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using System.Collections.ObjectModel;
+using System.Globalization;
+using System.IO;
+using System.Management.Automation;
+using System.Management.Automation.Runspaces;
+
+using YamlDotNet;
+using YamlDotNet.Serialization;
+using YamlDotNet.Serialization.NamingConventions;
+
+namespace Microsoft.PowerShell.PlatyPS
+{
+    /// <summary>
+    /// Cmdlet to generate the markdown help for commands, all commands in a module or from a MAML file.
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "YamlMetadata", HelpUri = "https://go.microsoft.com/fwlink/?LinkID=2096483")]
```

Comment:
The help URI is not correct.

Reply:
@adityapatwardhan  I talked to sean, he said they should all be empty strings for now